rpcs3/Emu/SysCalls/Modules/cellSpurs.cpp

```diff
@@ -191,7 +191,7 @@ s32 cellSpursEventFlagGetTasksetAddress(vm::ptr<CellSpursEventFlag> eventFlag, v
 //
 // SPURS lock free queue functions
 //
-s32 _cellSpursLFQueueInitialize();
+s32 _cellSpursLFQueueInitialize(vm::ptr<CellSpurs> spurs, vm::ptr<CellSpursTaskset> taskset, vm::ptr<CellSpursLFQueue> lfQueue, vm::cptr<void> queueBuffer, u32 queueSize, u32 queueDepth, u32 queueDirection);
```

Comment:
This function has only 6 arguments. And it seems you looked in a wrong place for its arguments.